hastory-api/src/Hastory/API.hs

```diff
@@ -74,6 +74,11 @@ mkHastoryClient url username password = do
         Left err -> pure $ Left err
         Right token -> pure $ Right (HastoryClient clientEnv token)
 
+mkUnauthenticatedClient :: MonadIO m => BaseUrl -> m ClientEnv
+mkUnauthenticatedClient url = liftIO (acceptManager <$> newManager defaultManagerSettings)
```

Comment:
You'll need to use `tlsManagerSettings` otherwise we cannot use `https` for the server.